docs/source/tutorials/object-detector.md

```diff
@@ -99,7 +104,7 @@ If you have a GPU available, you can specify that with `device=gpu:0` when you s
    device=gpu:0
    ```
 
-#### Bounding box thresholding
+#### Bounding box threshold
```

Comment:
it should be "thresholding", because it's a process applied over bounding boxes. 
Or "threshold bounding box's scores" where threshold is a verb